arkane/gaussian.py

```diff
@@ -466,6 +473,20 @@ def load_scan_frozen_atoms(self):
         """
         return self._load_scan_specs('F')
 
+    def load_scan_angle(self):
+        """
+        Return the angle difference (degrees) for a gaussian scan.
+        """
+        output = self._load_scan_specs('S', get_after_letter_spec=True)
+        return float(output[0][1])
+
+    def load_number_scans(self):
```

Comment:
I think we can assume a 360 scan (we do it anyway, right?) and only keep one of these methods, e.g., derive the angle diff from 360 / number of scans.

Reply:
After this PR, I am going to make another PR that no longer assumes a 360 degree scan. I think there are cases where scans may be shorter (like a methyl scan with 120 degrees) giving similar accuracy with 1/3rd the computation time. Ideally we should allow users this flexibility. The code is already written to implement the more robust scanning, I just wanted to make changing the hindered rotor treatment a separate PR (since this one is long enough and the addition would require different testing).